OpenAdaptAI/OpenAdapt
# fix: windows.get_active_window()

**What kind of change does this PR introduce?**

This PR changes the _windows.get_active_window_state() in https://github.com/OpenAdaptAI/OpenAdapt/blob/main/openadapt/window/_windows.py#L31 , to get the top window of the active application. This PR resolves getting the "No windows of that application are active" logger wawrning.

**Summary**

This PR is a bugfix for #331.

**Checklist**
* [x] I have perfomed a self-review of my code

**How can your code be run and tested?**

Create a new recording on windows, switch to a different application. You should not see the "No windows of that application are active" warning again.
